router/batchrouter/asyncdestinationmanager/sftp/types.go

```diff
@@ -0,0 +1,26 @@
+package sftp
+
+import (
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	sftp "github.com/rudderlabs/rudder-go-kit/sftp"
+)
+
+// DefaultManager is the default manager for SFTP
+type DefaultManager struct {
+	FileManager sftp.FileManager
+	logger      logger.Logger
+}
+
+type destConfig struct {
+	AuthMethod string `json:"authMethod"`
+	Username   string `json:"username"`
+	Host       string `json:"host"`
+	Port       string `json:"port"`
```

Comment:
```suggestion
	Port       int `json:"port"`
```
Suggesting changing to int to avoid converting the string back to int again

Reply:
JSON.unmarhsal is not giving the desired output for string to int conversion.